irohad/ametsuchi/impl/rocksdb_command_executor.cpp

```diff
@@ -53,6 +55,23 @@ RocksDbCommandExecutor::RocksDbCommandExecutor(
       vm_caller_{vm_caller},
       db_transaction_(db_context_) {
   assert(db_context_);
+
+  getSubscription()->dispatcher()->repeat(
+      SubscriptionEngineHandlers::kMetrics,
+      std::chrono::seconds(5ull),
```

Comment:
isn't better to have global variable instead of its?

Reply:
It doesn't use anywhere else. So it's better to add comment instead of global ns pollution.

Reply:
IMO Comment wouldn't help, because 5 seconds is understandable:)